src/main/kotlin/Creation.kt

```diff
@@ -0,0 +1,84 @@
+import Archive.Companion.archives
+import Input.Companion.scanner
+
+class Creation {
+    companion object {
+        val menu: Menu = Menu()
+        var archiveNumber: Int? = null
+
+        fun createArchive(): Archive {
+            println("введите название нового архива")
+            val name = scanner.nextLine()
+            println("архив $name успешно создан")
+            return Archive(name)
+        }
+
+        fun chooseArchive() {
+            if (archives.isNotEmpty()) {
+                println("выберите архив или назад")
+                for (archive in archives) {
+                    println("${archives.indexOf(archive)}. ${archive.title}")
+                }
+
+
+                val input = scanner.nextLine()
+
+                if (Input.checkInt(input)) {
+                    archiveNumber = input.toInt()
+                    if (input.toInt() >= 0 && input.toInt() <= archives.size) {
+                        menu.noteMenuPrint()
+                    }
+                }
+            } else {
+                println("Нет созданных архивов")
+            }
+        }
+
+        fun createNote() {
+            println("Создание заметки, введите название")
+            val title = scanner.nextLine()
+            println("Введите текст заметки")
+            val text = scanner.nextLine()
+            val note = Note(title, text)
+            putNoteToArchive(note)
+            println("Заметка $title успешно создана и добавлена в архив ${archives[archiveNumber!!].title}")
+            menu.noteMenuPrint()
+        }
+
+        fun putNoteToArchive(note: Note) {
+            archives[archiveNumber!!].notes.add(note)
+        }
+
+        fun chooseNote() {
+            if (archives[archiveNumber!!].notes.isNotEmpty()) {
```

Comment:
🍏 Использование оператора !! создает риск падения приложения в том случае, если в будущем порядок вызова методов будет изменен. Лучше сделать проверку на null. Например,
```suggestion
archiveNumber?.let {
...
}
```